pkg/server/endpoints/node/limiter_test.go

```diff
@@ -0,0 +1,141 @@
+package node
```

Comment:
nit: use github.com/stretchr/testify helpers to cut down on assertion boilterplate

Reply:
Done